packages/gatsby/src/commands/develop.ts

```diff
@@ -150,6 +150,7 @@ async function startServer(program: IProgram): Promise<IServer> {
   /**
    * Set up the express app.
    **/
+
```

Comment:
very NIT, but please avoid white space changes  (same for white space changes in query-runner.js)